projects/ad738x-st/builds.json

```diff
@@ -0,0 +1,7 @@
+{
+  "stm32": {
+    "basic_example": {
```

Comment:
instead of basic example, rename to iio_example

Reply:
ok, not sure what the CI was looking for in this file..

Reply:
well, you have the platform (stm32), the name of the example (iio_example) and then the compilation flags
the name of the binary in the release file will use these values, the final result being:
nameoftheprojectfolder_platform_nameoftheexample
so basically in your case it would be
ad738x-st_stm32_basic_example for a binary which is not the standard example, that is why it is better to use ad738x-st_stm32_iio_example to make it clear that this example uses iio.